helpers/mobileLogin.ts

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+import { Alert } from 'react-native';
+
+export default async function mobileLogin(email, password) {
+  const response = await axios.post(
+    'https://api-wanderlust-dogs.herokuapp.com/users/login/mobile',
```

Comment:
Can we change this? We should probably remove any database URL from our code and use environment variables.

Ctrl + Shift + F to search through the entire folder for wanderlust-dogs.

Reply:
yes

Reply:
tomorrow